database/migrations/2024_01_21_182512_add_flag_support_simple_zip_upload.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use App\Models\Extensions\BaseConfigMigration;
+
+return new class() extends BaseConfigMigration {
+	public const CONFIG = 'Image Processing';
+	public const BOOL = '0|1';
+	public const STRING = 'string';
+
+	public function getConfigs(): array
+	{
+		return [
+			[
+				'key' => 'extract_import_zip_files',
```

Comment:
The name of this config variable seems overly long and confusing. Could it simply be:

```suggestion
				'key' => 'extract_zip_files',
```

or:

```suggestion
				'key' => 'import_extract_zip_files',
```

Reply:
Fixed [5545adc](https://github.com/LycheeOrg/Lychee/pull/2229/commits/5545adc45401babc343acdcfc9007929a91b16b7).